tests/contrib/uwsgi/__init__.py

```diff
@@ -0,0 +1,20 @@
+import os
+import subprocess
+import sys
+
+
+def run_uwsgi(cmd):
+    def _run(*args):
+        env = os.environ.copy()
+        if sys.version_info[0] == 2:
+            # On Python2, it's impossible to import uwsgidecorators without this hack
```

Comment:
Do we have any info/link to why this is necessary?

Reply:
For both of these I'd have to defer to @jd who added this code for testing the profiler.